src/nvim/runtime.c

```diff
@@ -20,6 +21,8 @@
 # include "runtime.c.generated.h"
 #endif
 
+static bool search_path_valid = false;
+static RuntimeSearchPath runtime_search_path;
```

Comment:
would be nice to model this as a struct, pseudocode:

```
static rtp_searchpath = {
    RuntimeSearchPath path,
    bool valid,
}
```

in general I don't understand why we don't always do this as a habit, for all kinds of "familes" of globals.

Reply:
it is already scoped tho: to the file. A translation unit is already a "namespace", don't need a inner platform for that.

Reply:
aren't the variables related? At least a common prefix would help discover that, if not a structural grouping.

Reply:
originally they were but then windows CI threw a fit so I changed one to fix the problem and then they were no more.